.github/workflows/code_verify.yaml

```diff
@@ -21,7 +25,7 @@ jobs:
           go-version: 1.24.x
 
       - name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
I remember that @Monokaix reverted the v4 to v3 again: https://github.com/volcano-sh/volcano/pull/4251, could we upgrade this version now?